TesseractCore/Exporting.swift

```diff
@@ -0,0 +1,24 @@
+//  Copyright (c) 2015 Rob Rix. All rights reserved.
+
+private func concat(strings: [String]) -> String {
+    return reduce(strings, "", +)
+}
+
+public func export<T>(graph: Graph<T>) -> String {
+    var result = "digraph tesseract {\n"
+    result += concat(map(graph.edges, { edge in "\t" + edge.source.identifier.description + " -> " + edge.destination.identifier.description + ";\n" }))
+    result += "}"
+    return result
+}
+
+public func exportToFile<T>(graph: Graph<T>, filename: String) {
+    let result = export(graph)
+    let filehandle = NSFileHandle(forWritingAtPath: filename)
+    if let data = result.dataUsingEncoding(NSUTF8StringEncoding) {
+        filehandle?.writeData(data)
+    }
+}
```

Comment:
Upon reflection, I think I’d prefer to keep file handling out of this library altogether—it needn’t concern itself with path access, `URL` vs `String`, etc. I think `export()` above will suffice for that purpose.